spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -160,9 +139,7 @@ public class MyCredentials implements TokenCredentialProvider {
     }
```

Comment:
How does the user code know the credential of which keyvault to return when there are multiple?